lib/user_agent/browsers/amazon_echo.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class UserAgent
+  module Browsers
+    # Echo/1.0(APNG)
+    # Echo/1.0(MRM)
+    class AmazonEcho < Base
```

Comment:
What does the default parser do? I would expect it to work but have a wrong version number, but given the lack of a space between 1.0 and the comment, maybe not.

Reply:
Yeah, it is using the base parser and return the version with the comment too. I guess we have to regex only the version part.

Reply:
Okay, so this parser is basically so that `speaker?` works as expected? I was just trying to verify that it wasn't something else that might be a parser bug :)